wee_slack.py

```diff
@@ -2113,6 +2113,14 @@ def __init__(self, message_json, team, channel, override_sender=None):
                 message_json['text']
             )
```

Comment:
The same API change removed the `file_share` subtype, so the block above can be removed.